src/linux/backend-v4l2.cpp

```diff
@@ -841,7 +857,7 @@ namespace librealsense
             {
                 try
                 {
-                    if (!(cur_node.first.uvc_capabilities & V4L2_CAP_META_CAPTURE))
+                    if (! ((cur_node.first.uvc_capabilities & V4L2_CAP_META_CAPTURE) | (cur_node.first.mi == 3 && cur_node.first.uvc_capabilities & V4L2_CAP_VIDEO_CAPTURE_MPLANE)))
```

Comment:
Bitwise OR, should be Logic OR `||`

Reply:
Please add a comment about `cur_node.first.mi == 3`
So that readers can understand why you only allow 3